src/renderer.rs

```diff
@@ -530,7 +530,8 @@ impl Renderer {
 
         {
             // Present screen framebuffer to screen.
-            let mut p = f.pass(PassOp::Clear(Rgba::TRANSPARENT), present);
+            let background_appearance_color = Rgba::from(session.settings["background"].color());
+            let mut p = f.pass(PassOp::Clear(background_appearance_color), present);
```

Comment:
So this sets the background color of the session, not the views?

Reply:
Correct, this code changes the entire session UI appearance to that color, not the image views individually.

As you mentioned in the issue ticket, I think the "checker" option for transparency works pretty well for the use case of editing images with black pixels, so I'm not sure how useful it would be to set the individual views background color, at least for how I use it.

In my use case I tend to stretch the UI of this program to fill some large chunk of my 1920x1080 monitor and staring at a solid black background during daytime is a bit harsh after a few hours, hence my thinking for changing the whole session color :)

If you'd prefer I implement a way to change the view backgrounds, I suppose I could always do it that way.